### PR TITLE
Adicionando traducao da strings.xml para PT/BR - Add PT/BR translatio…

### PR DESCRIPTION
**Pull request by mxrpxcx**
#462

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -20,33 +20,33 @@
...
-    <string name="share_text_plant" translation_description="The default messaged used when sharing a plant on SNS, etc.">Check out the %s plant in the Android Sunflower app</string>
+    <string name="menu_filter_by_grow_zone" translation_description="The label for the menu item to filter the displayed plants by grow zone.">Filtrar por zona de germinação</string>
+    <string name="plant_detail_image_content_description" translation_description="Content description for the header / hero image that is show at the top of the plant detail screen.">Imagem de uma Planta</string>
+    <string name="my_garden_title" translation_description="Title for the 'My garden' screen where the user can see the list of plants that have been added to their garden.">Meu jardim</string>
+    <string name="plant_list_title" translation_description="Title for the plant list screen where the user can see a list of plants available to add to their garden.">Lista de Plantas</string>
+    <string name="available" translation_description="Text for the header of the plant list page indicating the available plants for users to choose from.">Plantas disponíveís</string>
```
Comment (gianei) [inline app/src/main/res/values/strings.xml]: There is a wrong additional accent. It should be `Plantas disponíveis`

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -20,33 +20,33 @@
...
+    <string name="plant_details_title" translation_description="Title for the plant details screen where the user can see the details for a specific plant.">Detalhes da Planta</string>
+    <string name="add_plant" translation_description="Button text that prompts user to navigate to the Plant List to add plants to their empty garden.">Adicionar uma Plantat</string>
```
Comment (gianei) [inline app/src/main/res/values/strings.xml]: `Adicionar uma Planta`

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -20,33 +20,33 @@
...
-    <string name="watering_needs_prefix" translation_description="Prefix for the text describing how often a plant should be watered. Used in combination with watering_needs_suffix.">Watering needs</string>
+    <string name="watering_needs_prefix" translation_description="Prefix for the text describing how often a plant should be watered. Used in combination with watering_needs_suffix.">Precisa ser regada</string>
     <plurals name="watering_needs_suffix">
-        <item quantity="one" translation_description="Suffix for the text describing how often a plant should be watered, singular. Used in combination with watering_needs_prefix: 'Watering needs: every day'.">every day</item>
-        <item quantity="other" translation_description="Suffix for the text describing how often a plant should be watered, plural. Used in combination with watering_needs_prefix: 'Watering needs: every 8 days'.">every %d days</item>
+        <item quantity="one" translation_description="Suffix for the text describing how often a plant should be watered, singular. Used in combination with watering_needs_prefix: 'Watering needs: every day'.">todo diay</item>
```
Comment (gianei) [inline app/src/main/res/values/strings.xml]: `todo dia`

**Review by gianei — CHANGES_REQUESTED** [review]

**Comment by riggaroo** [comment]
Hi, 
Thanks for your contribution, however Sunflower is going to be archived in favour of maintaining a smaller set of up-to-date set of compose-samples, that can be found here: https://github.com/android/compose-samples, as well as snippets that can be found here: https://github.com/android/snippets.

Please feel free to keep your fork up-to-date for your learning, but this repository will be archived shortly.

**Pull request closed without merging**